apis/bulkapproveapi/bulkapproveapi.yaml

```diff
@@ -0,0 +1,670 @@
+openapi: 3.0.0
+info:
+  title: Bulk_Approve
+  termsOfService: 'https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE'
+  description: These api's are all about the bulk approval of the contents and other scenarios included in that.
```

Comment:
@santoshilimi can you clarify what you mean by 'other scenarios included in that'?

Reply:
Please check,